src/cljs/reviews_next/pages/reviewEvent.cljs

```diff
@@ -0,0 +1,56 @@
+(ns reviews-next.pages.reviewEvent
```

Comment:
In clojure, we avoid camel-case names like this. 
The convention is use kebab-case/hyphenated names. The file names are usually snake case/underscored